dub.sdl

```diff
@@ -29,9 +29,8 @@ subPackage {
     "src/ddmd/tokens.d" \
     "src/ddmd/utf.d"
 
-  // stringImportPaths cannot be used because it will make Dub extremely slow
-  // https://github.com/dlang/dub/issues/1199
-  dflags "-J$PACKAGE_DIR"
+  preGenerateCommands "\"$${DUB_PACKAGE_DIR}config.sh\" \"$${DUB_PACKAGE_DIR}generated/dub\" VERSION /etc"
+  stringImportPaths "generated/dub"
```

Comment:
Well there is a difference between passing raw paths directly to dmd and telling dub which subfolder to use in a package. And yes it will work since dub knows how to deal with relative paths for the latter.